### ISSUE TITLE
Create users from client

### ISSUE THREAD

**Issue by derek-adair** [TryGhost/Ghost-CLI#1772]
### Summary
I would like to be able to add staff users from the command line.

`ghost createuser --type {contributor/author/editor/admin}`

**Comment by daniellockyer** [comment]
I think it's unlikely we'll add this as Ghost-CLI is a tool for managing the Ghost install. You can add users via the Admin/API 🙂

**Comment by derek-adair** [comment]
I found the admin API after i submitted this which is how I resolved my problem which prompted this request.  That makes a lot of sense regarding this being an install tool Vs the API being the way to modify/seed data.

On a tangential yet related subject; would it be useful to add some site config prompts to the install process?  It's definitely not a big deal for *my* preferences/workflow... but I would certainly prefer to skip the form when you access `/ghost`.  Being able to enter the following in during an install would be my preference;

1. site name
2. full name
3. email
4. password

I am indifferent on the subject -- just an idea.  Everything but site name is the same for me as I admin the sites that I install.
